status-im/status-legacy
# [Fix 8635] About showed "Version ()" instead of Version 0.14.0 (2019…)

fixes #8635 

### Summary

This fixes #8635 by adding VERSION and BUILD_NUMBER files in the correct nix template and updating bash script in order to prevent it from failing, due to the git repository being not initialized in the nix environment.

#### Platforms


- Android
- iOS
- macOS
- Linux
- Windows

### Steps to test
- Open Status
- Create an account
- Open Profile -> About and check value shown for Version

status: ready
